sympy/vector/coordsysrect.py

```diff
@@ -152,7 +152,7 @@ def __new__(cls, name, location=None, rotation_matrix=None,
 
         # Assign a Del operator instance
         from sympy.vector.deloperator import Del
-        obj._delop = Del(obj)
+        obj._delop = Del()
 
```

Comment:
We can remove `obj._delop` from constructor, but I leave it because of compatibility. We can add warnigs, when someone will be create `delop` from `CoordSysCartesian`